horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/RunServiceTest.java

```diff
@@ -59,24 +75,72 @@
    EventBus eventBus;
 
    @org.junit.jupiter.api.Test
-   public void testDataSetCreated(TestInfo info) throws InterruptedException {
+   public void testStraightforwardTransformationEmptyResult(TestInfo info) throws InterruptedException {
+      BlockingQueue<DataSet> dataSetQueue = eventConsumerQueue(DataSet.class, DataSet.EVENT_NEW);
+      NamedJsonPath path = createExampleNamedJsonPath("foo", "$.value");
+      Schema schema = createExampleSchema(info, true);
+
+      assertNotNull(schema.id);
+      Transformer transformer = createExampleTransformerWithJsonPath("acme", schema, path);
```

Comment:
I wonder if it's useful to create the schema and transformer if it can't be used. Yes, it rules out the case that 'some random' schema would be used accidentally - I wonder if that check is necessary.

Reply:
It's worth having. Avoid objects being produced when the schema is missing in the run data..